tutorials/src/test/scala/org/apache/daffodil/tutorials/TestTutorials.scala

```diff
@@ -26,7 +26,7 @@ object TestTutorials {
   val runner2 = Runner("/", "test1.tutorial.tdml.xml")
   val runner3 = Runner("/", "testTDMLTutorialFile2.tutorial.tdml.xml")
```

Comment:
Maybe only semi related to this change, but no tests are run using the runner3. Should that be delete along with the testTDMLTutorialFile? I'm not sure if that file should has a purpose or if it's just cruft. Also, what is test1.tutorial.tdml? Is that a tutorial or testing the tutorial capabilities? Should it be in src/test/ or maybe it just needs a more descriptive name other than test1?